transactions/transStringTemplates.py

```diff
@@ -0,0 +1,12 @@
+
+class TransactionsStringsTemplates:
+    contract_expiration_msg = (
+        "We would like to notify you that your contract with the **{franchise}** (team: {team}) franchise has expired."
+        "\n\n**So what does this mean?**\n\nAs a result, you are now a **Free Agent**, and you may sign to any franchise that "
+        "reaches out with an offer. You may re-sign to the same franchise if they offer you a new contract, or you may sign to "
+        "any other offering franchise of your choice. If you do not accept any offers prior to the deadlines in place, you will "
+        "become a Draft Eligible (DE) player, and will be drafted to a team. For more information, you may use the `{p}dates` "
+        "command in the #bot-input channel."
+
+        "\n\nIf you have any questions or concerns, feel free to reach out to your previous GM ({gm}), or open a mod-mail."
```

Comment:
We call this ModMail everywhere, not mod-mail. 🤷 Silly and pedantic, perhaps.

Reply:
Not at all, consistency is important!